pages/contributors/submission.vue

```diff
@@ -1,64 +1,66 @@
 <template>
   <v-layout justify-center column align-content-center>
     <h1>{{ $t('contributors.submission.title') }}</h1>
-    <woudc-blurb :items="blurb1" />
-    <span class="note">
-      <p>
-        <b>{{ $t('note') }} (1)</b>
-        {{ $t('contributors.submission.note1') }}
-      </p>
-    </span>
+    <p>{{ $t('contributors.submission.blurb1') }}</p>
+    <v-card class="woudc-note">
+      <v-card-text>
+        <strong>{{ $t('note') }} (1)</strong> {{ $t('contributors.submission.note1') }}
+      </v-card-text>
+    </v-card>
     <h2>{{ $t('contributors.submission.subtitle') }}</h2>
     <ol>
-      <li v-for="(blurb, i) in steps" :key="i">
-        <woudc-blurb :items="blurb" />
+      <li>
+        <i18n path="contributors.submission.step1.template" tag="span">
+          <template v-slot:formats>
+            <nuxt-link
+              :to="localePath('about-formats')"
+              v-text="$t('contributors.submission.step1.formats')"
+            />
+          </template>
+        </i18n>
       </li>
+      <li>
+        <i18n path="contributors.submission.step2.template" tag="span">
+          <template v-slot:ftp>
+            <a
+              :href="ftpPath"
+              target="_blank"
+              v-text="$t('contributors.submission.step2.ftp')"
+            />
+          </template>
+        </i18n>
+      </li>
+      <li>{{ $t('contributors.submission.step3') }}</li>
+      <li>{{ $t('contributors.submission.step4') }}</li>
+      <li>{{ $t('contributors.submission.step5') }}</li>
     </ol>
-    <span class="note">
-      <p>
-        <b>{{ $t('note') }} (2)</b>
-        {{ $t('contributors.submission.note2') }}
-      </p>
-    </span>
-    <woudc-blurb :items="blurb2" />
+    <v-card class="woudc-note">
+      <v-card-text>
+        <strong>{{ $t('note') }} (2)</strong> {{ $t('contributors.submission.note2') }}
+      </v-card-text>
+    </v-card>
+    <i18n path="contributors.submission.blurb2.template" tag="p">
+      <template v-slot:contact>
+        <nuxt-link
+          :to="localePath('contact')"
+          v-text="$t('contributors.submission.blurb2.contact')"
+        />
```

Comment:
Really? I had being specifically making all the `nuxt-link`s self-closing the whole way through. Since it affects files all across the project, this would be a good thing to go in the follow-up pull request I mention at https://github.com/woudc/woudc-ui/pull/5#discussion_r437521892.

Reply:
Yeah, there are very few void elements. Off the top of my head, they are: `<br />`, `<meta />`, `<img />` and `<my-custom-components />` that aren't meant to have content.

`<nuxt-link>` actually supports content: `<nuxt-link>Go to some page</nuxt-link>`

Style guide link about self-closing tags here: https://vuejs.org/v2/style-guide/#Self-closing-components-strongly-recommended